android/engine/build.gradle

```diff
@@ -102,17 +105,23 @@ dependencies {
     implementation 'androidx.paging:paging-runtime-ktx:3.0.1'
 
     //CQL
-    implementation "javax.xml.stream:stax-api:1.0-2"
-    implementation ("stax:stax:1.2.0") {
-        transitive = true
-        exclude group: 'stax', module: 'stax-api'
-    }
-    implementation "org.opencds.cqf.cql:engine:1.5.2"
-    def cqlEvaluatorVersion = '1.2.1-SNAPSHOT'
-    implementation 'org.opencds.cqf.cql:evaluator:' + cqlEvaluatorVersion
+    implementation group: 'xerces', name: 'xercesImpl', version: '2.11.0'
+    implementation(group: "com.github.java-json-tools", name: "msg-simple", version: "1.2");
+    implementation(group: "com.github.java-json-tools", name: "jackson-coreutils", version: "2.0");
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.12.2'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.2'
+    implementation 'com.fasterxml.jackson.core:jackson-annotations:2.12.2'
+    implementation 'org.codehaus.woodstox:woodstox-core-asl:4.4.1'
+    implementation 'javax.xml.stream:stax-api:1.0-2'
+    implementation "ca.uhn.hapi.fhir:hapi-fhir-android:5.4.0"
```

Comment:
Maybe check step by step which of these libraries we either already have or don't need for the CQL evaluation feature to work

e.g. you can run `./gradlew :engine:dependencies` command from the **android** folder in fhircore

Reply:
@ndegwamartin The libraries added are all need for CQL Evaluation especially during runtime

Reply:
@kimigxfoxy for example when I run the above shared command i.e. `./gradlew :engine:dependencies` I see that the jackson dependencies already exist in the dependency tree?

Reply:
@ndegwamartin I have removed the Jackson dependencies in the CQL section and tested that CQL evaluate in app works